backend/gameplay/consumers.py

```
@@ -1002,4 +1006,223 @@ async def draw_declined(self, event):
+				"type": "challenge_received",
+				"challenge_sender": self.scope["user"].username,
+				"relationship": relationship,
+				"challenge_time_control": challenge_time_control
+			}
+		)
+
+	async def accept_challenge(self, data):
+		challenge_sender = data["challenge_sender"]
+		challenge_obj: GameChallenge | None = await GameChallenge.async_get_challenge_from_sender_username(challenge_sender)
+
+		if challenge_obj == None:
```

Ruff (0.11.9)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)